README.md

```diff
@@ -15,17 +15,17 @@ Clone the repository from: [https://github.com/AlexAltea/cetrainer-unpacker](htt
 Run the following command with administrator privileges:
 
 ```
-extract.py path/to/trainer.exe [path/to/trainer.xml]
+extract.py path/to/trainer.exe [path/to/trainer.ct]
 ```
 
-Note that the second argument is optional and defaults to: `path/to/trainer.exe.xml`.
+Note that the second argument is optional and defaults to: `path/to/trainer.exe.ct`.
 
 
 ## Files
 
 ### `extract_dynamic.py`
 
-Extracts the `*.CETRAINER` file via binary instrumentation with [Frida](http://www.frida.re/). Specifically, it spawns the trainer and re-injects into its child processes until it intercepts file management calls to `*.CETRAINER` files. Then, if necessary, it will decrypt it to the plaintext (i.e. unprotected) XML version.
+Extracts the `*.CETRAINER` file via binary instrumentation with [Frida](http://www.frida.re/). Specifically, it spawns the trainer and re-injects into its child processes until it intercepts file management calls to `*.CETRAINER` files. Then, if necessary, it will decrypt it to the plaintext (i.e. unprotected) CT version.
```

Comment:
No, the format is still XML, we are only changing the extension out of convenience.

Reply:
Ok tks you very much :D